### PR TITLE
Change CRD to none-namespaced

### PR DESCRIPTION
**Pull request by yaocw2020**
issue: https://github.com/harvester/harvester/issues/582

### CODE REVIEW

File: pkg/controller/common/nodenetwork.go [hunk 1]
```diff
@@ -24,11 +23,10 @@ func NewNodeNetworkFromNode(node *corev1.Node, networkType networkv1.NetworkType
 	cn *networkv1.ClusterNetwork) *networkv1.NodeNetwork {
 	nn := &networkv1.NodeNetwork{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      node.Name + "-" + string(networkType),
-			Namespace: Namespace,
+			Name: node.Name + "-" + string(networkType),
 			OwnerReferences: []metav1.OwnerReference{
 				{
-					APIVersion: "ctlcorev1",
+					APIVersion: "core",
```
Comment (gitlawr) [inline pkg/controller/common/nodenetwork.go:29]: Should it be `v1`? that's what we get from the Kubernetes core APIs

Comment (yaocw2020) [reply pkg/controller/common/nodenetwork.go:29]: Yes，it should be v1. It has been always wrong before.

**Review by gitlawr — APPROVED** [review]
LGTM

**Comment by guangbochen** [comment]
@futuretea does either node driver or the terraform provider use the `clusterNetwork` or `nodeNetwork` resource?

**Comment by futuretea** [comment]
> @futuretea does either node driver or the terraform provider use the `clusterNetwork` or `nodeNetwork` resource?

`clusterNetwork`  was used by terraform provider

**Comment by yaocw2020** [comment]
dependency: https://github.com/harvester/harvester/issues/923

**Review by guangbochen — APPROVED** [review]
Will merge after https://github.com/harvester/harvester/issues/923

**Pull request merged**